tests/unit/trace/ext/httplib/test_httplib_trace.py

```diff
@@ -76,7 +76,7 @@ def test_wrap_httplib_request(self):
         method = 'GET'
 
         with patch:
-            wrapped(mock.Mock(), method, url)
```

Comment:
it will be really cool beyond the unit test have some functional test with actual httplib doing actual POST and GET requests.

Reply:
This may be a separate work item though